pytorch-cpu-requirements.txt

```diff
@@ -1,5 +1,4 @@
---pre
---index-url https://download.pytorch.org/whl/test/cpu
+--index-url https://download.pytorch.org/whl/cpu
```

Comment:
Might want to be careful with this `whl/test/cpu` index downstream too, @marbre : https://github.com/nod-ai/shark-ai/blob/c4a592ac8bcb2202a554ab1a4d311fdf5ddf28eb/.github/workflows/ci-sharktank.yml#L77